src/tlf.h

```diff
@@ -149,7 +149,7 @@ typedef struct {
     char call[20];
     int countrynr;
     char continent[3];
-    char cqzone[3];
+    unsigned short cqzone;
```

Comment:
I'd just use a plain `int`, even if wasting some bytes.

Reply:
yes, we also had countrynr as int